packages/forms/frontend/sirius-components-widget-reference/src/ReferencePropertySection.tsx

```diff
@@ -236,55 +206,16 @@ export const ReferencePropertySection = ({
   const onReferenceValueSimpleClick = (item: GQLReferenceValue) => {
```

Comment:
Are you removing the execution of some behavior on a click in the reference widget just like that or did I missed something? We can't simply remove a feature because you feel people are not using it. If you want to add some behavior to the reference widget, you should allow the specification of additional behavior not simply remove one you don't like. A shape and/or an ADR would be appropriate for that.

Reply:
No, I changed the behavior of the reference widget.
Today, specifiers can declare a set of operations (the body) on a reference widget (like most of other widgets). These operations are executed on a click on a selected item in the reference widget.
The click action is not used and we can't provide a custom behavior on set/add operations.
So this PR modify this behavior, by executing the body on set/add actions instead of the click one.
If no body operations are provided, then the default (i.e. the current) behavior on set/add operations will apply.